### DISCUSSION TITLE
Jimmer架构图修改意见

### DISCUSSION THREAD

**Discussion by babyfish-ct** [babyfish-ct/jimmer#442] [Ideas]
基础版本在此，欢迎大家提出自己的想法

![14a9630c08b37348d513bd73f566f574](https://github.com/babyfish-ct/jimmer/assets/15168900/bcd4b51d-8f73-47fd-9778-a1771a56f750)

**Comment by WackyGem** [comment]
1. 建议调整 Jimmer Entity 、及 Generated DTO Types 这两根长条状的图例占比。
2. 建议将 https://babyfish-ct.gitee.io/jimmer-doc/docs/resource/ecosystem 中的社区用户开发的辅助工具作为生态纳入架构图。

**Comment by pot-mot** [comment]
Entity部分可以提及一下interface多继承、属性可空性等特性。并且感觉可以考虑略去Generate DTO type，把这一块集中到API去。API部分可以更大一些，把DTO作为REST API部分的支撑。

**Comment by qinkangdeid** [comment]
过滤器、拦截器等也可以展示一个方块出来  因为这两块特性能够给予用户自定义拓展的大杀器
